aspnetcore/security/authentication/configure-jwt-bearer-authentication/sample/JwtBearer/Program.cs

```diff
@@ -0,0 +1,67 @@
+
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace JwtBearer;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        builder.Services.AddControllers();
+        builder.Services.AddOpenApi();
+
+        builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+            .AddJwtBearer(jwtOptions =>
+            {
+                jwtOptions.Authority = "https://{--your-authority--}";
+                jwtOptions.Audience = "https://{--your-audience--}";
+            });
+
+        builder.Services.AddAuthentication()
+            .AddJwtBearer("some-scheme", jwtOptions =>
+            {
+                jwtOptions.MetadataAddress = builder.Configuration["Api:MetadataAddress"]!;
+                jwtOptions.Authority = builder.Configuration["Api:Authority"];
+                jwtOptions.Audience = builder.Configuration["Api:Audience"];
+                jwtOptions.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = true,
+                    ValidateAudience = true,
+                    ValidateIssuerSigningKey = true,
+                    ValidAudiences = builder.Configuration.GetSection("Api:ValidAudiences").Get<string[]>(),
+                    ValidIssuers = builder.Configuration.GetSection("Api:ValidIssuers").Get<string[]>()
+                };
+
+                jwtOptions.MapInboundClaims = false;
+                jwtOptions.TokenValidationParameters.ValidTypes = ["at+jwt"];
+            });
+
+        var requireAuthPolicy = new AuthorizationPolicyBuilder()
+            .RequireAuthenticatedUser()
+            .Build();
+
+        builder.Services.AddAuthorizationBuilder()
+            .SetFallbackPolicy(requireAuthPolicy);
```

Comment:
```suggestion
```

We already using `MapControllers().RequireAuthorization()` and `[Authorize]`. It's worth documenting how to set a fallback policy which we already do above, but I don't think we need it in the main sample. I have seen it cause too many issues with Blazor and `[AllowAnonymous]` on specific pages because the fallback policy still gets applied to the static Blazor JS resources.